Vungle/VungleRouter.h

```diff
@@ -7,6 +7,7 @@
 
 #import <Foundation/Foundation.h>
 #import <VungleSDK/VungleSDK.h>
+#import <VungleSDK/VungleSDKHeaderBidding.h>
```

Comment:
Could move the header file to  "VungleRouter.m"  file, thanks. :)

Reply:
Resolved.